pkg/config/config.go

```diff
@@ -75,6 +75,12 @@ func (c *Config) InitConfig() {
 		c.ProfilesFile = configFile
 		viper.SetConfigType("toml")
 		viper.SetConfigFile(configFile)
+		viper.SetConfigPermissions(os.FileMode(0600))
+
+		// Try to change permissions manually, because we used to create files
+		// with default permissions (0644). Ignore errors (e.g. if the file
+		// doesn't exist yet).
+		_ = os.Chmod(configFile, os.FileMode(0600))
```

Comment:
Is this consistent on non-unixes (aka Windows)?

Reply:
Also instead of swallowing the error, can we swallow the "file-non-existant" subset of this https://golang.org/src/os/error.go?s=1314:1377#L30 just in-case there is some other permissions related error that the user may need help with

Reply:
For Windows:
> On Windows, only the 0200 bit (owner writable) of mode is used; it controls whether the file's read-only attribute is set or cleared. The other bits are currently unused. For compatibility with Go 1.12 and earlier, use a non-zero mode. Use mode 0400 for a read-only file and 0600 for a readable+writable file.